kindlecomicconverter/comic2ebook.py

```diff
@@ -1222,6 +1224,58 @@ def checkPre(source):
         raise UserWarning("Target directory is not writable.")
 
 
+def makeFusion(sources, qtgui=None):
+    filepath = []
+    GUI = qtgui
+    start = perf_counter()
+    pageTracker : int = 0
+    combinePath : str = os.path.join(os.path.dirname(sources[0]), "combinationTemp")
+    os.mkdir(combinePath)
+    print("Running Fusion")
+        
+    for source in sources:
+        print(f"Processing {source}...")
+        
+        checkPre(source)
+        print("Checking images...")
+        path : str = getWorkFolder(source)
+        filepath.append(path)
+        pathfinder = (os.path.join(path, "OEBPS", "Images"))
+        print(pathfinder)
+        images = sorted(os.listdir(pathfinder))
+        for image in images:
+            pageTracker += 1
+            ext = os.path.splitext(image)[1]
+            target_path = os.path.join(combinePath, f"{pageTracker}{ext}")
+            os.rename(os.path.join(pathfinder, image), target_path)
+
+        
+    #Ouput
+    sanitizeTree(combinePath)
+    print("Creating CBZ file...")
+    for paths in filepath:
+        if os.path.isfile(paths):
+            os.remove(paths)
+        elif os.path.isdir(paths):
+            rmtree(paths)
+    filepath = makeZIP('Combined_comic', combinePath)
```

Comment:
zipping the final output is unneccessary, KCC can take as input nested folders of images. And the name should be the first source's file name, plus something at the end like `fused`